components/routes/Drawers.js

```diff
@@ -24,6 +25,15 @@ export const DefaultDrawer = () => {
               />
             )
           })} />
+        <Drawer.Screen name="Daily Rides" component={DailyRidesScreen}
```

Comment:
This works and is a good way of testing the screen for now.
However, we can go ahead and place this screen on the home screen.

Steps to do this:
1. Place a button on the Home screen (It doesn't need to rotate or be dynamic right now)
2. Build a Stack navigator (`Stack.js`) in the `routes` folder. You can check out the example [here](https://reactnavigation.org/docs/stack-navigator/#example). Also, the Stack navigator will look very similar to the `Drawer.js` and `Tab.js` files already in the codebase.
3. Create a stack within the newly created `Stack.js` and call it `HomeStack`. You will just need to place the Home screen and the Daily Ride screens in this stack. 
4. Once the stack is created you can replace the `HomeScreen` in the `HomeTab` with your new `HomeStack`.
5. Finally have the button you placed on the home screen navigate to the Daily Rides screen. This can be done using `navigation.navigate('DailyRidesScreen')`. See [here](https://reactnavigation.org/docs/navigation-events/) for information about this.

If you want, we can work on this together during our meeting tomorrow.

Reply:
Great lineup of to-do items, @CalebMcOlin ! I will have them listed out as separate issues and will implement in separate PR(s), per my comment above